beta/css/discovery.css

```diff
@@ -138,11 +138,14 @@ footer {
 
 footer ul {
   padding-inline-start: 0;
+  padding: 8px 0px;
```

Comment:
When I changed the layout to get the error message displaying correctly I'd taken this out so that the new layout matched how you designed it (on the desktop).